custom_metrics/sass.js

```diff
@@ -189,6 +189,11 @@ function analyzeSCSS(scss, ret) {
 		"class": scss.match(/&\.\w+/g)?.length,
 		"attr": scss.match(/&\[\w+/g)?.length,
 	};
+
+	ret.constants = {
+		e: scss.match(/\bmath\.$e\b/g)?.length,
```

Comment:
Does the `$` need to be escaped?

Reply:
Oops, yes! Oh well, I suppose no point fixing it now.